libraries/lib-channel/Channel.h

```diff
@@ -87,7 +87,8 @@ class CHANNEL_API WideChannelGroupInterval : public ChannelGroupInterval {
        result: `!(iChannel < NChannels()) || result`
     */
    template<typename IntervalType = ChannelInterval>
-   std::shared_ptr<IntervalType> GetChannel(size_t iChannel)
+   std::enable_if_t<std::is_base_of_v<ChannelInterval, IntervalType>, std::shared_ptr<IntervalType>>
+   GetChannel(size_t iChannel)
```

Comment:
Maybe this change isn't needed.  It rules out cases where `static_cast` below would not work but `dynamic_cast` would work.

Note too that there is a `const` overload that you did not similarly change.

Reply:
`const` version resolves to non-const, which could be SFINAE'd, but I removed this change from PR completely